app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> str:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
+
+        vaccine_info = visitor["vaccine"]
+        expiration_date = vaccine_info["expiration_date"]
+
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError()
+
+        if not visitor.get("wearing_a_mask", False):

```

Comment:
Code Efficiency: There is no need to provide a default value of `False` to the `get` method. If the key is not found in the dictionary, `get` will return `None` which is considered `False` in a boolean context.